episodes/chapter4.md

```diff
@@ -0,0 +1,244 @@
+
+
+## Manage vantage6 project using the UI
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+Before starting this section, make sure that the participants have enough permissions to create a new organization, a new user, and a new collaboration. If not, give them the necessary permissions.
+Note that the role `Collaboration Admin` does not have the permission to create a new organization.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+In this section, we will go through the steps to create a new organization, a new user, a new collaboration and reset an api key for a node using the vantage6 UI.
+
+First you need to log in to the vantage6 UI, and then go to the `Administration` page. You can find the `Administration` button in the top right corner of the start page.
+
+![vantage6 UI administration page](fig/ui_admin_page.png)
+
+### Create a new organization
+It's quite straightforward to create a new organization in vantage6. Here are the steps:
+
+1. Click on the `Organizations` tab in the administration page.
+2. Click on the `Create organization` button.
+3. Fill in the details of the new organization.
+    * The `Upload public key` field is optional. You can upload a public key for the organization if you want to use encryption in the collaboration.
+4. Click on the `Submit` button to create the new organization.
+
+![Create a new organization](fig/create_org.png)
+
+### Create a new user
+Now let's create a new user for the organization we just created. Here are the steps:
+
+1. Click on the `Users` tab in the administration page.
+2. Click on the `Create user` button.
+3. Fill in the details of the new user.
+    * You can assign the user to an organization by selecting it from the `Organization` dropdown. Only one organization can be selected.
+    * You can assign roles to the user by selecting them from the `Roles` dropdown. Here we assign the `Researcher` role to the new user. You can give the user more permissions by assigning multiple roles or select the operation boxes in the `Permissions` section.
+4. Click on the `Submit` button to create the new user.
+
+![Create a new user](fig/create_user.png)
+
+### Create a new collaboration
+A collaboration is a collection of one or more organizations. Let's create a new collaboration between two organizations. Here are the steps:
+
+1. Click on the `Collaborations` tab in the administration page.
+2. Click on the `Create collaboration` button.
+3. Fill in the details of the new collaboration.
+    * For `Encrypted` box, you can select whether the collaboration should be encrypted or not.
+    * You can select the organizations that will participate in the collaboration by selecting them from the `Organizations` dropdown.
+    * By default, we select the `Register nodes` box. This will ensure the nodes of the organizations are registered in the collaboration. If you don't select this box, you will have to register the nodes manually later.
```

Comment:
I think this part should make clear that this action is only the first step in creating a collaboration, and that enabling it needs participant nodes to register their corresponding keys on a v6-node. For learning purposes, it could also elaborate more on what the keys are for, and how are they related to the ER diagram above: each organization has a unique key for each collaboration; if an organization has multiple collaborations, it needs multiple nodes, each one with its corresponding key.

I think the ideal way of 'closing' this episode would be for the participants to set up a collaboration with working nodes and run an algorithm like they did in episode 3. However, that would require that somehow they register the keys on the nodes configuration. @CunliangGeng and @bartvanb just to open the discussion, as editing YAML files on a server terminal may be too advanced for this module, maybe this can be done by having a person or process dedicated to setup 'dummy' data nodes as needed by the participants' collaborations (who would simple provide the keys they got in the process)?

Reply:
The chapter 4 targets at newbie, focusing on the UI. The setup of v6 node is described in chapter 6. I can add a few words here to redirect to chapter 6.